ironfish/src/wallet/account.ts

```diff
@@ -104,7 +116,7 @@ export class Account {
         continue
       }
 
-      if (!decryptedNote.index) {
+      if (!decryptedNote.sequence || decryptedNote.sequence > maxConfirmedSequence) {
```

Comment:
nit: Can we make this its own condition? It's kind of a style thing but I think it helps clearly understand each of the cases. Here we say if it's not on the chain, skip it, then next  we check that its in the confirmation range.

Reply:
That sounds good to me! I'll make the change